static/app/views/dashboards/hooks/useValidateWidget.tsx

```diff
@@ -24,8 +24,15 @@ export function useValidateWidgetQuery(_widget: Widget) {
   const organization = useOrganization();
   const {selection} = usePageFilters();
 
+  const cleanedWidget = cleanWidgetForRequest(_widget);
+
+  // Pin title and description to avoid re-triggering validation on
+  // every change in title/description.
+  cleanedWidget.title = 'sentinel';
```

Comment:
Didn't really know what to put here, so I used `sentinel` to mean it's not really supposed to do anything and indicate some kind of placeholder.